Cargo.toml

```diff
@@ -115,7 +115,7 @@ base64 = "0.21.0"
 blake2-rfc = { version = "0.2.18", default-features = false }
 bs58 = { version = "0.4.0", default-features = false }
 clap = { version = "4.2.1" }
-codec = { package = "parity-scale-codec", version = "3.4.0", default-features = false }
+codec = { package = "parity-scale-codec", version = "3.6.4", default-features = false }
```

Comment:
Maybe remove this shortcut at all?

Reply:
@breathx look at https://github.com/gear-tech/gear/pull/2925. gshep did `git cherry-pick` on the substrate repository. Maybe it's still useful to leave at least in the node?

Reply:
Feel free to just replace it there.
Right, @gshep?

Reply:
it's possible to keep the shortcut as well, but we might need another PR to clean our workspace, like declaring

```
codec = { package = "parity-scale-codec", workspace = true }
```

in each of the packages using it

ref https://github.com/paritytech/subxt/pull/932